airflow/www/templates/airflow/dags.html

```diff
@@ -19,6 +19,7 @@
 
 {% extends base_template %}
 {% from 'appbuilder/loading_dots.html' import loading_dots %}
+{% from 'appbuilder/dataset_next_run_modal.html' import dataset_next_run_modal %}
```

Comment:
This file should probably be 
```suggestion
{% from 'airflow/dataset_next_run_modal.html' import dataset_next_run_modal %}
```